Add JS Framework : Rexuws

Aderinom opened the pull request:
Hello Community! 

I found a express adapter for uWebsockets.js, Rexuws.
https://github.com/rexuws/rexuws

I want to see how well this does in the benchmarks,to see if I should start putting some work into it. 

With this PR I want to add it to the JS benchmarks.

waghanza commented on javascript/rexuws/config.yaml:
```diff
@@ -0,0 +1,14 @@
+framework:
+  website: https://github.com/rexuws/rexuws
```
not useful (website will target github due to next config)

waghanza commented on javascript/rexuws/config.yaml:
```diff
@@ -0,0 +1,14 @@
+framework:
+  website: https://github.com/rexuws/rexuws
+  github: rexuws/rexuws
+  version: 1.1.4
```
```suggestion
  version: 1.1
```

waghanza commented on javascript/rexuws/package.json:
```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "rexuws": "https://github.com/Aderinom/rexuws-update-deps"
```
a fork ?

Aderinom replied:
Yes. Updated the version of uws to allow newer node version to run it. 
Didn't really think about the ergononics at that point though.

Already read your comment. Makes sense to block it until then.

waghanza commented:
I only can accept this `PR` after https://github.com/rexuws/rexuws/pull/27 is released @Aderinom 

However, I appreciate your participation, and feel free to contribute for anything else (there is some issue listing javascript frameworks, like `Adonis`)

Aderinom commented:
Pushed a possible workaround for the incompability of the older rexuws version.
Added the newer UWS version as a override. Allows the server to run under Node 18.14.

If that's too hacky I understand. 

Also changed the config.yaml as you requested.

waghanza commented on javascript/rexuws/app.js:
```diff
@@ -0,0 +1,32 @@
+});
+
+app.listen(3000, function () {
+  console.log("listening on port 3000");
```
I think this could mess-up the results a little bit

Aderinom replied:
This only runs once after the socket has successfully been bound, not on every request.

So this should not cause any difference in req/s.

Aderinom replied:
Buuut... since it's not required I guess I'll remove it

waghanza commented on javascript/rexuws/package.json:
```diff
@@ -0,0 +1,14 @@
+        "run": "node app.js"
+    },
+    "dependencies": {
+        "rexuws": "^1.1.4",
```
```suggestion
        "rexuws": "~1.1.4",
```

the idea is to have the same operation type in all package.json for javascript (not sure which is more standard)

Aderinom replied:
Sure, makes sense to limit it to only minor version changes.

waghanza reviewed (changes requested):
let's wait a little bit to merge this (maybe in the next round), just to make sure the author accept https://github.com/rexuws/rexuws/pull/27

waghanza commented:
I'll close this PR, and rise it again when `rexuws` 2.x will be out

Aderinom commented:
Yes, sorry. I completely forgot about this!
